src/api/v1.js

```diff
@@ -69,4 +70,22 @@ apiV1Router.get('/getFullPuzzle', async (req, res) => {
     }
 });
 
+apiV1Router.post('/createPuzzleSet', async (req, res) => {
+    const required = ['name', 'order'];
+
+    const missing = required.filter((prop) => {
```

Comment:
How about `some`/`every`?

Reply:
To collect missing fields and report them back to the user